lambda_layers/core_layer/python/core_layer/handler/item_handler.py

```diff
@@ -155,12 +165,22 @@ def get_open_items_for_user(user, num_items, session) -> Dict[List[Item], bool]:
             return {'items': items, 'is_open_review': False}
 
     # Get open items for junior review and return them
-    result = session.query(Item) \
+    query_base = session.query(Item) \
         .filter(Item.open_reviews_level_1 > Item.in_progress_reviews_level_1) \
         .filter(~Item.reviews.any(Review.user_id == user.id)) \
-        .filter(Item.status == "open") \
-        .order_by(Item.open_timestamp.asc()) \
-        .limit(num_items).all()
+        .filter(Item.status == "open")
+
+    oldest_items = query_base.order_by(Item.open_timestamp.asc()) \
+        .limit(math.floor(num_items/2)).all()
+
+    latest_items = query_base.order_by(Item.open_timestamp.desc()) \
+        .limit(15) \
+        .from_self() \
+        .order_by(Item.open_reviews_level_1.asc(), Item.open_timestamp.desc()) \
+        .limit(math.floor(num_items/2 + 1)) \
+        .all()
```

Comment:
Könnte man die Logik nicht in einzelne Methoden auslagern und somit die Code Duplikation zu weiter oben vermeiden?

Reply:
Erledigt. Kannst du bitte nochmal kurz draufschauen?